sdk/notificationhubs/notification-hubs/src/api/beginSubmitNotificationHubJob.ts

```diff
@@ -117,30 +120,45 @@ export async function beginSubmitNotificationHubJob(
       return () => progressCallbacks.delete(s);
     },
 
-    isDone(): boolean {
+    get isDone(): boolean {
       return ["succeeded", "failed", "canceled"].includes(state.status);
     },
 
-    stopPolling(): void {
-      abortController.abort();
-      cancelJob?.();
+    get operationState(): OperationState<NotificationHubJob> | undefined {
+      return state;
     },
 
-    isStopped(): boolean {
-      return resultPromise === undefined;
+    get result(): NotificationHubJob | undefined {
+      return state.result;
     },
 
-    getOperationState(): OperationState<NotificationHubJob> {
-      return state;
+    async serialize(): Promise<string> {
+      return JSON.stringify({ state });
     },
 
-    getResult(): NotificationHubJob | undefined {
-      return state.result;
+    async submitted() {
```

Comment:
In core-lro we resolve submitted promise with two conditions 1) the initial request is back and 2) the poller is initialized(not `notStarted` status).

Reply:
If I understand correctly we could make this return true as soon as the initial response is sent?